### PR TITLE
Feature/config file

### PR DESCRIPTION
**Pull request by jnayak1**
Move reference to localhost:8000 to config file.

### CODE REVIEW

File: config/environment.js [hunk 1]
```diff
@@ -29,6 +29,7 @@ module.exports = function(environment) {
     // ENV.APP.LOG_TRANSITIONS = true;
     // ENV.APP.LOG_TRANSITIONS_INTERNAL = true;
     // ENV.APP.LOG_VIEW_LOOKUPS = true;
+    ENV.meetingsUrl = 'http://localhost:8000';
```
Comment (Ghalko) [inline config/environment.js:32]: Will it always be `development` for `localhost:8000`?

Comment (jnayak1) [inline config/environment.js:32]: I'm not sure if I understand exactly, but when ever we are in development mode (running locally), meetingsUrl will refer to `http://localhost:8000`. We can make test and production mode point to staging and osf.io respectively. @Ghalko

Comment (Ghalko) [inline config/environment.js:32]: This works with just `ember server`? @jnayak1

Comment (jnayak1) [inline config/environment.js:32]: Yes, are you having problems with it? @Ghalko 
http://stackoverflow.com/questions/29444229/how-to-make-a-production-ready-build-using-ember-cli
`ember build --environment=production` to setup production
`ember server` should cause `environment=development`

**Pull request merged**